### DISCUSSION TITLE
电子邮件应用程序的新外观

### DISCUSSION THREAD

**Discussion by Sonic1paRadise** [linuxdeepin/developer-center#3465] [Features Request & Ideas | 特性请求 & 头脑风暴]
![v23 mail template](https://user-images.githubusercontent.com/51707249/195523955-88918027-df7e-4570-b7dc-f32339782371.png)

**Comment by Sonic1paRadise** [comment]
另外，我们需要在电子邮件应用程序中添加某种文具
（请没有库存照片或剪贴画！！！）

**Reply by BLumia** [reply]
Hi, i don't understand what did you mean. If you can speak English, you can post English so we can probably better understand about your suggestions. Thanks!
